lib/datadog/tracing/trace_operation.rb

```diff
@@ -113,8 +113,22 @@ def finished?
         @finished == true
       end
 
+      # Will this trace be flushed by the tracer transport?
+      # This includes cases where the span is kept solely due to priority sampling.
```

Comment:
It is true that either `@sampled == true` OR `@sampling_priority > 0` means it will be flushed by the tracer.

I know that because that was the logic in `def sampled?` before I started this task :) And it also seems right: if there's any intention on keeping the trace, we should flush it.

We are very conservative today with flushing as much as possible, in order to allow the agent to calculate metrics, so any signal of willingness to keep the span is taken into account.

Reply:
Okay, I think re-reading this message, and the PR description, things make more sense now. Thanks for explaining.